addons/mrp/models/stock_move.py

```diff
@@ -274,6 +274,12 @@ def _onchange_product_uom_qty(self):
             new_qty = float_round((mo.qty_producing - mo.qty_produced) * self.unit_factor, precision_rounding=self.product_uom.rounding)
             self.quantity = new_qty
 
+    @api.onchange('quantity', 'product_uom', 'picked')
+    def _onchange_quantity(self):
+        if self.raw_material_production_id and not self.manual_consumption and self.picked and self.product_uom and \
```

Comment:
`self.product_uom` is a required field :thinking: do you need to check that it's set?

Reply:
Hello @ticodoo ! In a previous version of this fix, I did not include this check on the uom and then I realized that this breaks a lot of the mrp tests using forms:
https://runbot.odoo.com/runbot/build/62932160

The idea is that you can create "new" moves without products and hence with an empty "product_uom":
https://github.com/odoo/odoo/blob/663eb1fdb57f01f32ed791ce36478de2c65fb35a/addons/mrp/tests/test_bom.py#L1335-L1337